Source/TrackManagerUI.cpp

```
@@ -188,6 +208,48 @@ bool TrackManagerUI::isMinimapResizeCursorActive() const {
            m_timelineMinimap->getCursorHint() == NomadUI::TimelineMinimapCursorHint::ResizeHorizontal;
 }
 
+bool TrackManagerUI::isCustomCursorActive() const {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)